src/D2L.CodeStyle.Analyzers/Language/RequireNamedArgumentsAnalyzer.cs

```diff
@@ -9,12 +9,29 @@
 namespace D2L.CodeStyle.Analyzers.Language {
 	[DiagnosticAnalyzer( LanguageNames.CSharp )]
 	public sealed class RequireNamedArgumentsAnalyzer : DiagnosticAnalyzer {
+
+		public sealed class ArgParamBinding {
+			public ArgParamBinding(
+				int position,
+				string paramName,
+				ArgumentSyntax syntax
+			) {
+				Position = position;
+				ParamName = paramName;
+				Syntax = syntax;
+			}
+
+			public int Position { get; }
+			public string ParamName { get; }
+			public ArgumentSyntax Syntax { get; }
+		}
+
 		public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics => ImmutableArray.Create(
 			Diagnostics.UseNamedArgsForInvocationWithLotsOfArgs
 		);
 
 		// TODO: shrink this number over time. Maybe 5 would be good?
-		public const int TOO_MANY_UNNAMED_ARGS = 64;
+		public const int TOO_MANY_UNNAMED_ARGS = 63;

```

Comment:
Accidentally made this bigger than necessary in original commit.

Reply:
Not fixing in this PR because I'd have to update the tests and that's really unrelated.